clients/include/testing_dot_strided_batched.hpp

```diff
@@ -226,7 +226,7 @@ void testing_dot_strided_batched(const Arguments& arg)
         if(arg.norm_check)
         {
             std::cout << "cpu=" << cpu_result << ", gpu_host_ptr=" << rocblas_result_1
-                      << ", gpu_device_ptr=" << rocblas_result_2 << "\n";
+                      << ", gpu_device_ptr=" << rocblas_result_2 << std::endl;
```

Comment:
should this be rocblas_cout   rather than std::cout   ?

Reply:
Yes, that is part of the "mechanical changes", s/std::cout/rocblas_cout/. This PR left out those mechanical changes. std::cout is a poisoned identifier in rocblas-test and rocblas-bench with this PR, so if you use it, those won't compile (that poisoning does not affect outside users of rocBLAS such as samples; the poisoning is only in the rocblas-test / rocblas-bench source code, and not the library's).